src/sections/Community/community.style.js

```diff
@@ -179,8 +179,13 @@ const CommunitySectionWrapper = styled.div`
         .map {
             padding-top: 0rem;
             margin-top:0rem;
+            margin-left: 2rem;
```

Comment:
Looks good, only if you could change it to use the shorthand declaration for `margin`.